modules/manage/pages/kubernetes/storage/volume-types.adoc

```diff
@@ -0,0 +1,208 @@
+| High availability
+|===
+
+=== Reclaim policies
+
+PersistentVolumeClaims are not deleted when Redpanda brokers are removed from a cluster. It is your responsibility to delete PersistentVolumeClaims when they are no longer needed. As a result, it's important to consider your volume's reclaim policy:
+
+* *Delete*: The associated storage is removed once the PVC is deleted.
+* *Retain*: Even after the PVC is deleted, the actual storage persists. You can reclaim it with a new PVC.
+
+TIP: Considering the importance of data,  Redpanda Data recommends the `Retain` policy to ensure no accidental data loss.
+
+=== Persistent Volume Provisioners in Kubernetes
+
+Kubernetes offers two methods to provision Persistent Volumes (PVs): static provisioning and dynamic provisioning.
+
+==== Static provisioners
+
+Static provisioners require manual intervention from administrators. An administrator pre-provisions a set of PersistentVolumes. These volumes are physical storage spaces within the cluster that are ready for use but aren't bound to any specific PersistentVolumeClaims (PVCs) yet. When a PVC is created, the Kubernetes control plane looks for a PV that matches the requirements of the PVC and binds them together.
+
+When you use a static provisioner, You must create one PersistentVolume for each Redpanda broker. When the Redpanda Helm chart is deployed, an existing PersistentVolume in the cluster is selected and bound to one PersistentVolumeClaim for each Redpanda broker.
+
+[cols="1a,1a"]
+|===
+|Advantages|Disadvantages
+
+|Offers more control over the specific pieces of infrastructure that get attached to the cluster.
+|More manual work and overhead for the administrator.
+
+|Useful for non-cloud environments or for storage solutions that don't support dynamic provisioning.
+|Requires advanced planning of the size and number of PVs.
+|===
+
+==== Dynamic provisioners
+
+Dynamic provisioners eliminate the manual step of creating PersistentVolumes. Instead, a dynamic provisioner creates a PersistentVolume on demand for each Redpanda broker.
+
+This automatic provisioning is achieved using StorageClasses. A StorageClass provides a way for administrators to describe the classes of storage they offer. Each class can specify:
+
+- The type of provisioner to use
+- The type of storage backend
+- Other storage-specific parameters
+
+When the Redpanda Helm chart creates a PVC that specifies a StorageClass with a dynamic provisioner, Kubernetes automatically creates a new PV that matches the requirements of the PVC.
+
+Managed Kubernetes services and cloud environments usually provide a dynamic provisioner. If you are not using a managed service, you may need to install a dynamic provisioner for your storage type.
+
+[cols="1a,1a"]
+|===
+|Advantages|Disadvantages
+
+|Reduces manual administrative tasks. It's easy to scale up the storage as the PVs are created on-the-fly.
```

Comment:
|Reduces manual administrative tasks. It's simpler to scale up the storage as the PVs are created on-the-fly.